packages/maskbook/src/utils/theme-tools.ts

```diff
@@ -27,18 +29,46 @@ export function shade(channels: RGB, percentage: number): RGB {
     return channels.map((c) => clamp(Math.floor((c * (100 + percentage)) / 100), 0, 255)) as RGB
 }
 
+function fromRGBAtoRGB(color: string): string {
+    const matched = color.match(/^rgba\((\d{1,3}%?),\s*(\d{1,3}%?),\s*(\d{1,3}%?),\s*(\d*(?:\.\d+)?)\)$/)
+    if (matched) {
+        const [_, r, g, b, a] = matched
+        const rgba: RGBA = [parseInt(r), parseInt(g), parseInt(b), parseInt(a)]
+
+        return toRGB(shade(rgba.slice(0, 3) as RGB, rgba[3] * 100))
+    }
+
+    return ''
+}
+
+function isRGBA(color: string) {
+    return color.match(/^rgba\((\d{1,3}%?),\s*(\d{1,3}%?),\s*(\d{1,3}%?),\s*(\d*(?:\.\d+)?)\)$/)
+}
+
 export function getBackgroundColor(element: HTMLElement | HTMLBodyElement) {
     const color = getComputedStyle(element).backgroundColor
+    if (isRGBA(color)) {
```

Comment:
The colors in minds.com are rgba. Please tell me if u have a better idea than this implementation